### DISCUSSION TITLE
command in "Under the hood of " doesn't work

### DISCUSSION THREAD

**Discussion by bigcat1001** [milaboratory/mixcr#972] [General]
Hi mixcr team:
     I tried to adjust some parameters according to my  scheme, so I found the "Under the hood pipeline" of "SMARTer Human BCR IgG IgM H/K/L Profiling Sequencing Kit" and prepared to modify it  to fit my scheme. 
     However, the first error displays "No preset with name" kAligner2_4.0 " ". This means that we can only modify the parameters under the existing preset, right?

**Comment by mizraelson** [comment]
Hi, 
We have renamed some of the basic presets, I will fix it in the docs. The correct one here would be
`bundle-umi-kaligner2-v1-base`.

Also, if you can tell more about the modifications you want to make - I can help you with that. A lot of things can be modified directly from the analyze command. If there are a lot of adjustments you want to make I can help you with creating a specific preset for your needs so you wont have to provide those every time you run mixcr.

**Reply by bigcat1001** [reply]
Hi mizraelson
    Thanks for your enthusiasm. I am trying to understand the meaning of some parameters of MIXCR. If I encounter more problems, I will ask you for further help.
    I tried “bundle-umi-kaligner2-v1-base“. However, it showed "Preset bundle-umi-kaligner2-v1-base is abstract and not intended to be used directly".My MIXCR version is V4.1.2 and academic license.

**Comment by mizraelson** [comment]
Sorry, my bad, we did make those bundles abstract at the end, thus you can only inherit from it but not use as a separate preset.

So there are three options:
1) You can create your own preset based on the one we have for SMARTer Human BCR IgG IgM H/K/L Profiling Sequencing Kit and add some modifications to it.
2) You can use mix-ins to modify the preset.

If you give me an example of any parameter you wanna change I will show an example of how you can do that.

Also, you can just use:
3) mixcr align -p takara-human-bcr-cdr3 

and add some parameters, and they will override the ones in the preset.
